tensorflow/lite/micro/kernels/micro_ops.h

```diff
@@ -39,6 +39,8 @@ TfLiteRegistration Register_DEPTHWISE_CONV_2D();
 TfLiteRegistration Register_ELU();
 TfLiteRegistration Register_EXP();
 TfLiteRegistration Register_EXPAND_DIMS();
+TfLiteRegistration Register_L2_POOL_2D();
+TfLiteRegistration Register_LEAKY_RELU();
```

Comment:
This looks like an unrelated fix for Leaky Relu? Could you add it to a separate PR?

Reply:
There was a merge conflict resolution error.  Fixed now.